src/main/java/org/iot/dsa/dslink/dynamodb/DynamoDBDSAClient.java

```diff
@@ -0,0 +1,146 @@
+package org.iot.dsa.dslink.dynamodb;
+
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
+import com.amazonaws.services.dynamodbv2.document.DynamoDB;
+import com.amazonaws.services.dynamodbv2.model.ListTablesResult;
+
+import org.iot.dsa.node.DSMap;
+
+import java.util.List;
+
+public class DynamoDBDSAClient {
+
+    private String accessKey = null;
+    private String secretKey = null;
+    private String region = null;
+    private String endpoint = null;
```

Comment:
It seems these fields are not used, should they be removed?

Reply:
done. Removed!